### DISCUSSION TITLE
How to get all refs?

### DISCUSSION THREAD

**Discussion by xstxhjh** [vuejs/rfcs#348] [General]
dev
```js
// It exists
<script setup>
import { getCurrentInstance, onMounted } from "vue";

const { ctx } = getCurrentInstance();
onMounted(()=>{
  console.log(ctx.$refs)})
})
</script>
```

won't do
build or playground
https://sfc.vuejs.org/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8aDEgcmVmPVwibXlEb21cIj5IZWxsbyBXb3JsZCE8L2gxPlxuPC90ZW1wbGF0ZT5cblxuPHNjcmlwdCBzZXR1cD5cbmltcG9ydCB7IGdldEN1cnJlbnRJbnN0YW5jZSwgb25Nb3VudGVkLCByZWYgfSBmcm9tIFwidnVlXCI7XG5jb25zdCBteURvbSA9IHJlZihudWxsKVxuXG5jb25zdCB7IGN0eCB9ID0gZ2V0Q3VycmVudEluc3RhbmNlKCk7XG5vbk1vdW50ZWQoKCk9PntcbiAgY29uc29sZS5sb2coY3R4LiRyZWZzKVxuICBjb25zb2xlLmxvZyhteURvbSlcbn0pXG48L3NjcmlwdD4ifQ==

**Comment by Justineo** [comment]
See [here](https://sfc.vuejs.org/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8aDEgcmVmPVwidGl0bGVcIj5IZWxsbyBXb3JsZCE8L2gxPlxuPC90ZW1wbGF0ZT5cblxuPHNjcmlwdCBzZXR1cD5cbmltcG9ydCB7IG9uTW91bnRlZCwgcmVmIH0gZnJvbSBcInZ1ZVwiO1xuY29uc3QgdGl0bGUgPSByZWYobnVsbClcblxub25Nb3VudGVkKCgpPT57XG4gIGNvbnNvbGUubG9nKHRpdGxlLnZhbHVlKVxufSlcbjwvc2NyaXB0PiJ9).

**Reply by xstxhjh** [reply]
I know. How to find all refs?
